test/unit/adapters/http.js

```diff
@@ -160,7 +157,7 @@ describe('supports http with nodejs', function () {
       axios.get('http://localhost:4444/').then(function (res) {
         assert.deepEqual(res.data, data);
         done();
-      });
+      }).catch(done);
```

Comment:
If there's an error, `done` should be called with it so Mocha knows it's a failure.